src/Microsoft.AspNetCore.Mvc.Testing/build/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -0,0 +1,41 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We do but through a direct import, due to the fact that target files are only automatically imported for package references, we'll have some E2E tests for this on the identity repo.